docs/activities.md

```diff
@@ -139,7 +139,7 @@ In most cases, we recommend monitoring the `temporal_activity_schedule_to_start_
 
 A Start-To-Close Timeout is the maximum time allowed for a single [Activity Task Execution](/tasks#activity-task-execution).
 
-- [How to set a Start-To-Close Timeout](/application-development/features#start-to-close-timeout)
+- [How to set a Start-To-Close Timeout](/go/how-to-set-a-start-to-close-timeout-in-go)
```

Comment:
Need to change the concept file to point to "app-dev-context/activity-retries" so that linkMagic can do its work here.